frontend/src/app/signUp/Organization/OrganizationForm.tsx

```diff
@@ -0,0 +1,223 @@
+      await isFieldValid({ data: organization, schema, errors, field })
+    );
+  };
+
+  const getValidationStatus = (field: keyof OrganizationCreateRequest) =>
+    errors[field] ? "error" : undefined;
+
+  const onSave = async () => {
+    setLoading(true);
+    const validation = await isFormValid({
+      data: organization,
+      schema,
+    });
+    if (validation.valid) {
+      try {
+        await SignUpApi.createOrganization(organization);
+      } catch (error) {
+        const alert = (
+          <Alert type="error" title="Submission Error" body={error} />
+        );
+        showNotification(toast, alert);
+        setLoading(false);
+        return;
+      }
+      setErrors(initOrgErrors());
+      setSubmitted(true);
+      return;
+    }
+    setErrors(validation.errors);
+    const alert = (
+      <Alert
+        type="error"
+        title="Form Errors"
+        body="Please check the form to make sure you complete all of the required fields."
+      />
+    );
+    showNotification(toast, alert);
+    setLoading(false);
+  };
+
+  if (submitted) {
+    return <Thanks />;
+  }
+
+  if (loading) {
+    return <LoadingCard message="Creating organization" />;
+  }
+
+  const getFormElement = (
+    field: keyof OrganizationCreateRequest,
+    label: string | React.ReactNode,
+    required: boolean,
+    subheader: string | null
+  ) => {
+    switch (field) {
+      case "state":
+        return (
+          <Select
+            label={label}
+            name="state"
+            value={organization.state || ""}
+            options={stateCodes.map((c) => ({ label: c, value: c }))}
+            defaultSelect
+            onChange={onDetailChange("state")}
+            onBlur={() => {
+              validateField("state");
+            }}
+            validationStatus={getValidationStatus("state")}
+            errorMessage={errors.state}
+            required
+          />
+        );
+      case "type":
+        return (
+          <Select
+            label={label}
+            name="type"
+            value={organization.type || ""}
+            options={Object.entries(
+              OrganizationTypeEnum
+            ).map(([key, value]) => ({ label: value, value: key }))}
+            defaultSelect
+            onChange={onDetailChange("type")}
+            onBlur={() => {
+              validateField("type");
+            }}
+            validationStatus={getValidationStatus("type")}
+            errorMessage={errors.type}
+            required
+          />
+        );
+      default:
+        // This is not a field, just some explanatory text
+        if (label === "Organization administrator") {
+          return (
+            <>
+              <label className="usa-label margin-top-3">
+                <strong>{label}</strong>
+                <TextWithTooltip
+                  tooltip="Can be anyone that works at your organization. Adds testing facility locations and manages the organization account."
```

Comment:
**content question:** @Rebecca-LM I feel like we may want to mention something like "manages user access" to this org admin tooltip as well?

Reply:
@emmastephenson We can change the second sentence in the tooltip to: "Adds testing facility locations; manages the account and user access."